db/migrate/20240314225935_create_coursetoplatforms.rb

```diff
@@ -0,0 +1,10 @@
+class CreateCoursetoplatforms < ActiveRecord::Migration[7.1]
+  def change
+    create_table :coursetoplatforms do |t|
```

Comment:
```suggestion
    create_table :course_to_platform do |t|
```
SQL convention is to snake_case multi-word table names.